### PR TITLE
Fix the extra Spark config input issue with persistence

### PR DESCRIPTION
**Pull request by wezhang**
The Spark Remote Debug run configuration supports all paramters
persistence now.

Signed-off-by: Zhang Wei <zhwe@microsoft.com>

### CODE REVIEW

**Comment by wezhang** [comment]
```
[ERROR] Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'mars' from location http://download.eclipse.org/releases/mars: Unable to read repository at http://download.eclipse.org/releases/mars. HTTP Server 'Service Unavailable': http://download.eclipse.org/releases/mars/201506241002/content.xml.xz: HttpComponents connection error response code 503. -> [Help 1]
```

**Comment by wezhang** [comment]
Default Jenkins build hit:

```
Tests run: 13, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 40.444 sec <<< FAILURE! - in com.microsoft.tooling.msservices.serviceexplorer.azure.rediscache.RedisCacheCreateIntegrationTest
testRedisCacheCreateBasicWithExistedResGrp(com.microsoft.tooling.msservices.serviceexplorer.azure.rediscache.RedisCacheCreateIntegrationTest)  Time elapsed: 30.086 sec  <<< ERROR!
java.lang.Exception: Cannot create workMock instance
	at com.microsoft.tooling.IntegrationTestBase.setUpStep(IntegrationTestBase.java:113)
	at com.microsoft.tooling.msservices.serviceexplorer.azure.rediscache.RedisCacheCreateIntegrationTest.setUp(RedisCacheCreateIntegrationTest.java:132)

testRedisCacheCreateBasicWithExistedResGrp(com.microsoft.tooling.msservices.serviceexplorer.azure.rediscache.RedisCacheCreateIntegrationTest)  Time elapsed: 30.088 sec  <<< ERROR!
org.apache.http.conn.HttpHostConnectException: Connect to localhost:3122 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:141)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:107)
	at com.github.tomakehurst.wiremock.client.HttpAdminClient.safelyExecuteRequest(HttpAdminClient.java:311)
	at com.github.tomakehurst.wiremock.client.HttpAdminClient.postJsonAssertOkAndReturnBody(HttpAdminClient.java:261)
	at com.github.tomakehurst.wiremock.client.HttpAdminClient.resetAll(HttpAdminClient.java:144)
	at com.github.tomakehurst.wiremock.client.WireMock.resetMappings(WireMock.java:210)
	at com.microsoft.tooling.IntegrationTestBase.resetTest(IntegrationTestBase.java:158)
	at com.microsoft.tooling.msservices.serviceexplorer.azure.rediscache.RedisCacheCreateIntegrationTest.tearDown(RedisCacheCreateIntegrationTest.java:200)
```

@adashen , I've hit the RedisCacheCreateIntegrationTest related tesing failures in our Jenkins build serveral times. Could you help to have a look?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/uihelper/InteractiveRenderer.java [hunk 1]
```diff
@@ -35,16 +35,8 @@ public InteractiveRenderer(int interactiveColumn) {
...
         Component component =  super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column);
-        if (column == interactiveColumn && hasFocus) {
-
-            InteractiveTableModel tableModel = (InteractiveTableModel)table.getModel();
-            if(tableModel != null)
-
-            if ((tableModel.getRowCount() - 1) == row && !tableModel.hasEmptyRow())
-            {
-               tableModel.addEmptyRow();
-            }
```
Comment (djlan) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/uihelper/InteractiveRenderer.java]: I think one empty row is needed, so users can double click this row to add their customized row

**Comment by djlan** [comment]
:shipit:

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/uihelper/InteractiveRenderer.java (hunk 1 above)
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/uihelper/InteractiveRenderer.java]: Yes, we have added the empty row in the end of bulked adding.

**Pull request merged**